src/completed.js

```diff
@@ -0,0 +1,13 @@
+const checkBox = (index, items) => {
+  items[index].completed = !items[index].completed;
+  localStorage.localData = JSON.stringify(items);
+};
+
+export default function addCheckboxEvents(items) {
```

Comment:
- [] Please try to use the newer es6 syntax in situations where it is possible. It is recommended to use the newer arrow function syntax.
something like this would be great
```suggestion
export default (items) => {
```